docs/en/api/README.md

```diff
@@ -1,3 +1,3 @@
-## [mount](/docs/en/api/mount.md)
-## [shallow](/docs/en/api/shallow/README.md)
-## [selectors](/docs/en/api/selectors.md)
+## [mount](/api/mount.md)
```

Comment:
It would be even better to have `api/mount.md` (relative links). That way it'd be easier to build translations (and it's shorter 😆 )

Reply:
Oh yes I also thought about that, but I noticed the other links were all `/api/xxx.md`, 😆 
so I had to keep them consistent.

![image](https://user-images.githubusercontent.com/6647633/30004120-d9be671c-90fb-11e7-8fe6-e1eb57def210.png)

Reply:
I served the gitbook, and found it just worked, without problems.

Reply:
Seems the other links were all absolute paths, so I kept them consistent.

![image](https://user-images.githubusercontent.com/6647633/30004139-3944a8b8-90fc-11e7-9dcb-1aa9354cf0e9.png)